library/orientdb

```diff
@@ -1,15 +1,15 @@
 
 Maintainers: Luigi Dell'Aquila <l.dellaquila@orientdb.com> (@luigidellaquila)
 
-Tags: 3.2.8, 3.2, latest
+Tags: 3.2.8, 3.9, latest
```

Comment:
I don't think you meant to change this line in this specific way :sweat_smile:

Maybe this?

```suggestion
Tags: 3.2.9, 3.2, latest
```